archunit/src/main/java/com/tngtech/archunit/core/domain/JavaCodeUnit.java

```diff
@@ -171,6 +171,12 @@ public JavaClass getRawReturnType() {
         return fieldAccesses;
     }
 
+    @PublicAPI(usage = ACCESS)
+    public abstract Set<? extends JavaAccess<?>> getCallsOfSelf();
```

Comment:
Couldn't the bound here be `JavaCall` instead of `JavaAccess`?

Reply:
Changed.